project-1/test_gspread_code.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Oct 24 12:25:03 2023
+
+@author: computer world
```

Comment:
I understand this is a template line of code from Spyder or Pycharm... use the correct details, or just remove it.